### DISCUSSION TITLE
I can't get Hyprland to work

### DISCUSSION THREAD

**Discussion by higorlucas125** [hyprwm/Hyprland#4339] [General]
Guys, I need help, I can't get Hyprland to work, I've tried everything, I looked at the tutorial and everything, but Hyprland doesn't start, I'm using sddm with xcfe4 it works, but with hyprland it doesn't, I've looked at the sddm logs that these are

░░ Subject: A start job for unit sddm.service has finished successfully
░░ A start job for unit sddm.service has finished successfully.
Jan 03 10:19:18 arch2 sddm[674]: Initializing...
Jan 03 10:19:18 arch2 sddm[674]: Starting...
Jan 03 10:19:18 arch2 sddm[674]: Logind interface found
Jan 03 10:19:18 arch2 sddm[674]: Adding new display...
Jan 03 10:19:18 arch2 sddm[674]: Loaded empty theme configuration
Jan 03 10:19:18 arch2 sddm[674]: Xauthority path: "/run/sddm/xauth_JzCjTF"
Jan 03 10:19:18 arch2 sddm[674]: Using VT 2
Jan 03 10:19:18 arch2 sddm[674]: Display server starting...
Jan 03 10:19:18 arch2 sddm[674]: Writing cookie to "/run/sddm/xauth_JzCjTF"
Jan 03 10:19:18 arch2 sddm[674]: Running: /usr/bin/X -nolisten tcp -background none -seat seat0 vt2 -auth /run/sddm/xauth_JzCjTF -noreset -displayfd 16
Jan 03 10:19:18 arch2 sddm[674]: Setting default cursor
Jan 03 10:19:18 arch2 sddm[674]: Could not setup default cursor
Jan 03 10:19:18 arch2 sddm[674]: Running display setup script  "/usr/share/sddm/scripts/Xsetup"
Jan 03 10:19:18 arch2 sddm[674]: Display server started.
Jan 03 10:19:18 arch2 sddm[674]: Socket server starting...
Jan 03 10:19:18 arch2 sddm[674]: Socket server started.
Jan 03 10:19:18 arch2 sddm[674]: Loading theme configuration from "/usr/share/sddm/themes/simple-sddm/theme.conf"
Jan 03 10:19:18 arch2 sddm[674]: Greeter starting...
Jan 03 10:19:18 arch2 sddm-helper[705]: [PAM] Starting...
Jan 03 10:19:18 arch2 sddm-helper[705]: [PAM] Authenticating...
Jan 03 10:19:18 arch2 sddm-helper[705]: [PAM] returning.
Jan 03 10:19:18 arch2 sddm-helper[705]: pam_unix(sddm-greeter:session): session opened for user sddm(uid=969) by (uid=0)
Jan 03 10:19:19 arch2 systemd-logind[617]: New session c1 of user sddm.
░░ Subject: A new session c1 has been created for user sddm
░░ A new session with the ID c1 has been created for the user sddm.
Jan 03 10:19:19 arch2 (systemd)[709]: pam_warn(systemd-user:setcred): function=[pam_sm_setcred] flags=0x8002 service=[systemd-user] terminal=[] user=[sddm] ruser=[<unknown>] rhost=[<unknown>]
Jan 03 10:19:19 arch2 (systemd)[709]: pam_unix(systemd-user:session): session opened for user sddm(uid=969) by sddm(uid=0)
Jan 03 10:19:19 arch2 systemd[1]: Started Session c1 of User sddm.
Jan 03 10:19:19 arch2 sddm-helper[705]: Writing cookie to "/tmp/xauth_WeIDTc"
Jan 03 10:19:19 arch2 sddm-helper[705]: Starting X11 session: "" "/usr/bin/sddm-greeter --socket /tmp/sddm-:0-ROupqf --theme /usr/share/sddm/themes/simple-sddm"
Jan 03 10:19:19 arch2 sddm[674]: Greeter session started successfully
Jan 03 10:19:19 arch2 sddm-greeter[718]: High-DPI autoscaling Enabled
Jan 03 10:19:19 arch2 sddm-greeter[718]: Reading from "/usr/local/share/wayland-sessions/hyprland.desktop"
Jan 03 10:19:19 arch2 sddm-greeter[718]: Reading from "/usr/share/wayland-sessions/hyprland.desktop"
Jan 03 10:19:19 arch2 sddm-greeter[718]: Loading theme configuration from "/usr/share/sddm/themes/simple-sddm/theme.conf"
Jan 03 10:19:19 arch2 sddm-greeter[718]: Connected to the daemon.
Jan 03 10:19:19 arch2 sddm[674]: Message received from greeter: Connect
Jan 03 10:19:19 arch2 sddm-greeter[718]: Loading file:///usr/share/sddm/themes/simple-sddm/Main.qml...
Jan 03 10:19:19 arch2 sddm-greeter[718]: QFont::setPointSizeF: Point size <= 0 (0.000000), must be greater than 0
Jan 03 10:19:19 arch2 sddm-greeter[718]: file:///usr/share/sddm/themes/simple-sddm/Components/VirtualKeyboard.qml:26:1: module "QtQuick.VirtualKeyboard" is not installed
Jan 03 10:19:19 arch2 sddm-greeter[718]: file:///usr/share/sddm/themes/simple-sddm/Components/Input.qml:561:5: QML Connections: Implicitly defined onFoo properties in Connections are deprecated. Use this syntax instead: function onFoo(<arguments>) { ... }
Jan 03 10:19:19 arch2 sddm-greeter[718]: file:///usr/share/sddm/themes/simple-sddm/Components/SystemButtons.qml:70:13: Unable to assign [undefined] to QQuickItem*
Jan 03 10:19:19 arch2 sddm-greeter[718]: file:///usr/share/sddm/themes/simple-sddm/Main.qml:35: ReferenceError: Screen is not defined
Jan 03 10:19:19 arch2 sddm-greeter[718]: file:///usr/share/sddm/themes/simple-sddm/Components/SessionButton.qml:38:5: Unable to assign ComboBox_QMLTYPE_21 to Control_QMLTYPE_10
Jan 03 10:19:19 arch2 sddm-greeter[718]: file:///usr/share/sddm/themes/simple-sddm/Main.qml:34: ReferenceError: Screen is not defined
Jan 03 10:19:19 arch2 sddm-greeter[718]: Adding view for "eDP-1" QRect(0,0 1920x1080)
Jan 03 10:19:19 arch2 sddm-greeter[718]: Message received from daemon: Capabilities
Jan 03 10:19:19 arch2 sddm-greeter[718]: file:///usr/share/sddm/themes/simple-sddm/Components/SystemButtons.qml:70:13: Unable to assign [undefined] to QQuickItem*
Jan 03 10:19:19 arch2 sddm-greeter[718]: file:///usr/share/sddm/themes/simple-sddm/Components/SystemButtons.qml:70:13: Unable to assign [undefined] to QQuickItem*
Jan 03 10:19:19 arch2 sddm-greeter[718]: file:///usr/share/sddm/themes/simple-sddm/Components/SystemButtons.qml:70:13: Unable to assign [undefined] to QQuickItem*
Jan 03 10:19:19 arch2 sddm-greeter[718]: file:///usr/share/sddm/themes/simple-sddm/Components/SystemButtons.qml:70:13: Unable to assign [undefined] to QQuickItem*
Jan 03 10:19:19 arch2 sddm-greeter[718]: file:///usr/share/sddm/themes/simple-sddm/Components/SystemButtons.qml:70:13: Unable to assign [undefined] to QQuickItem*
Jan 03 10:19:19 arch2 sddm-greeter[718]: file:///usr/share/sddm/themes/simple-sddm/Components/SystemButtons.qml:70:13: Unable to assign [undefined] to QQuickItem*
Jan 03 10:19:19 arch2 sddm-greeter[718]: file:///usr/share/sddm/themes/simple-sddm/Components/SystemButtons.qml:70:13: Unable to assign [undefined] to QQuickItem*
Jan 03 10:19:19 arch2 sddm-greeter[718]: file:///usr/share/sddm/themes/simple-sddm/Components/SystemButtons.qml:70:13: Unable to assign [undefined] to QQuickItem*
Jan 03 10:19:19 arch2 sddm-greeter[718]: file:///usr/share/sddm/themes/simple-sddm/Components/SystemButtons.qml:70:13: Unable to assign [undefined] to QQuickItem*
Jan 03 10:19:19 arch2 sddm-greeter[718]: file:///usr/share/sddm/themes/simple-sddm/Components/SystemButtons.qml:70:13: Unable to assign [undefined] to QQuickItem*
Jan 03 10:19:19 arch2 sddm-greeter[718]: file:///usr/share/sddm/themes/simple-sddm/Components/SystemButtons.qml:70:13: Unable to assign [undefined] to QQuickItem*
Jan 03 10:19:19 arch2 sddm-greeter[718]: file:///usr/share/sddm/themes/simple-sddm/Components/SystemButtons.qml:70:13: Unable to assign [undefined] to QQuickItem*
Jan 03 10:19:19 arch2 sddm-greeter[718]: file:///usr/share/sddm/themes/simple-sddm/Components/SystemButtons.qml:70:13: Unable to assign [undefined] to QQuickItem*
Jan 03 10:19:19 arch2 sddm-greeter[718]: file:///usr/share/sddm/themes/simple-sddm/Components/SystemButtons.qml:70:13: Unable to assign [undefined] to QQuickItem*
Jan 03 10:19:19 arch2 sddm-greeter[718]: file:///usr/share/sddm/themes/simple-sddm/Components/SystemButtons.qml:70:13: Unable to assign [undefined] to QQuickItem*
Jan 03 10:19:19 arch2 sddm-greeter[718]: file:///usr/share/sddm/themes/simple-sddm/Components/SystemButtons.qml:70:13: Unable to assign [undefined] to QQuickItem*
Jan 03 10:19:19 arch2 sddm-greeter[718]: Message received from daemon: HostName
Jan 03 10:20:03 arch2 sddm-greeter[718]: Reading from "/usr/share/wayland-sessions/hyprland.desktop"
Jan 03 10:20:03 arch2 sddm[674]: Message received from greeter: Login
Jan 03 10:20:03 arch2 sddm[674]: Reading from "/usr/share/wayland-sessions/hyprland.desktop"
Jan 03 10:20:03 arch2 sddm[674]: Session "/usr/share/wayland-sessions/hyprland.desktop" selected, command: "Hyprland" for VT 1
Jan 03 10:20:03 arch2 sddm-helper[803]: [PAM] Starting...
Jan 03 10:20:03 arch2 sddm-helper[803]: [PAM] Authenticating...
Jan 03 10:20:03 arch2 dbus-daemon[615]: [system] Activating via systemd: service name='org.freedesktop.home1' unit='dbus-org.freedesktop.home1.service' requested by ':1.13' (uid=0 pid=803 comm="/usr/lib/sddm/sddm-helper --socket /tmp/sddm-auth-")
Jan 03 10:20:03 arch2 sddm-helper[803]: [PAM] Preparing to converse...
Jan 03 10:20:03 arch2 sddm-helper[803]: [PAM] Conversation with 1 messages
Jan 03 10:20:04 arch2 sddm-helper[803]: [PAM] returning.
Jan 03 10:20:04 arch2 sddm[674]: Authentication for user  "higorlucas"  successful
Jan 03 10:20:04 arch2 sddm-greeter[718]: Message received from daemon: LoginSucceeded
Jan 03 10:20:04 arch2 sddm-helper[803]: pam_unix(sddm:session): session opened for user higorlucas(uid=1000) by higorlucas(uid=0)
Jan 03 10:20:04 arch2 sddm-helper[705]: [PAM] Closing session
Jan 03 10:20:04 arch2 sddm-helper[705]: [PAM] Ended.
Jan 03 10:20:04 arch2 sddm-helper[705]: pam_unix(sddm-greeter:session): session closed for user sddm
Jan 03 10:20:04 arch2 sddm[674]: Auth: sddm-helper exited successfully
Jan 03 10:20:04 arch2 sddm[674]: Greeter stopped. SDDM::Auth::HELPER_SUCCESS
Jan 03 10:20:04 arch2 sddm-helper[803]: Starting Wayland user session: "/usr/share/sddm/scripts/wayland-session" "Hyprland"
Jan 03 10:20:04 arch2 sddm-helper[818]: Jumping to VT 1
Jan 03 10:20:04 arch2 sddm-helper[818]: VT mode didn't need to be fixed
Jan 03 10:20:04 arch2 sddm[674]: Session started true
Jan 03 10:20:06 arch2 sddm-helper[803]: [PAM] Closing session
Jan 03 10:20:06 arch2 sddm-helper[803]: pam_unix(sddm:session): session closed for user higorlucas
Jan 03 10:20:06 arch2 sddm-helper[803]: [PAM] Ended.
Jan 03 10:20:06 arch2 sddm[674]: Auth: sddm-helper exited with 6
Jan 03 10:20:06 arch2 sddm[674]: Socket server stopping...
Jan 03 10:20:06 arch2 sddm[674]: Socket server stopped.
Jan 03 10:20:06 arch2 sddm[674]: Display server stopping...
Jan 03 10:20:07 arch2 sddm[674]: Display server stopped.
Jan 03 10:20:07 arch2 sddm[674]: Running display stop script  ("/usr/share/sddm/scripts/Xstop")
Jan 03 10:20:07 arch2 sddm[674]: Removing display SDDM::Display(0x55fcf9720840) ...
Jan 03 10:20:07 arch2 sddm[674]: Adding new display...
Jan 03 10:20:07 arch2 sddm[674]: Loaded empty theme configuration
Jan 03 10:20:07 arch2 sddm[674]: Xauthority path: "/run/sddm/xauth_ptrnbt"
Jan 03 10:20:07 arch2 sddm[674]: Using VT 2
Jan 03 10:20:07 arch2 sddm[674]: Display server starting...
Jan 03 10:20:07 arch2 sddm[674]: Writing cookie to "/run/sddm/xauth_ptrnbt"
Jan 03 10:20:07 arch2 sddm[674]: Running: /usr/bin/X -nolisten tcp -background none -seat seat0 vt2 -auth /run/sddm/xauth_ptrnbt -noreset -displayfd 18
Jan 03 10:20:07 arch2 sddm[674]: Setting default cursor
Jan 03 10:20:07 arch2 sddm[674]: Could not setup default cursor
Jan 03 10:20:07 arch2 sddm[674]: Running display setup script  "/usr/share/sddm/scripts/Xsetup"
Jan 03 10:20:07 arch2 sddm[674]: Display server started.
Jan 03 10:20:07 arch2 sddm[674]: Socket server starting...
Jan 03 10:20:07 arch2 sddm[674]: Socket server started.
Jan 03 10:20:07 arch2 sddm[674]: Loading theme configuration from "/usr/share/sddm/themes/simple-sddm/theme.conf"
Jan 03 10:20:07 arch2 sddm[674]: Greeter starting...
Jan 03 10:20:07 arch2 sddm-helper[1044]: [PAM] Starting...
Jan 03 10:20:07 arch2 sddm-helper[1044]: [PAM] Authenticating...
Jan 03 10:20:07 arch2 sddm-helper[1044]: [PAM] returning.
Jan 03 10:20:07 arch2 sddm-helper[1044]: pam_unix(sddm-greeter:session): session opened for user sddm(uid=969) by (uid=0)
Jan 03 10:20:07 arch2 systemd-logind[617]: New session c2 of user sddm.
░░ Subject: A new session c2 has been created for user sddm
░░ A new session with the ID c2 has been created for the user sddm.
Jan 03 10:20:07 arch2 systemd[1]: Started Session c2 of User sddm.
Jan 03 10:20:07 arch2 sddm-helper[1044]: Writing cookie to "/tmp/xauth_QCmxId"
Jan 03 10:20:07 arch2 sddm-helper[1044]: Starting X11 session: "" "/usr/bin/sddm-greeter --socket /tmp/sddm-:0-RjjGUa --theme /usr/share/sddm/themes/simple-sddm"
Jan 03 10:20:07 arch2 sddm[674]: Greeter session started successfully
Jan 03 10:20:07 arch2 sddm-greeter[1045]: High-DPI autoscaling Enabled
Jan 03 10:20:07 arch2 sddm-greeter[1045]: Reading from "/usr/local/share/wayland-sessions/hyprland.desktop"
Jan 03 10:20:07 arch2 sddm-greeter[1045]: Reading from "/usr/share/wayland-sessions/hyprland.desktop"
Jan 03 10:20:07 arch2 sddm-greeter[1045]: Loading theme configuration from "/usr/share/sddm/themes/simple-sddm/theme.conf"
Jan 03 10:20:07 arch2 sddm-greeter[1045]: Connected to the daemon.
Jan 03 10:20:07 arch2 sddm[674]: Message received from greeter: Connect
Jan 03 10:20:07 arch2 sddm-greeter[1045]: Loading file:///usr/share/sddm/themes/simple-sddm/Main.qml...
Jan 03 10:20:07 arch2 sddm-greeter[1045]: QFont::setPointSizeF: Point size <= 0 (0.000000), must be greater than 0
Jan 03 10:20:07 arch2 sddm-greeter[1045]: file:///usr/share/sddm/themes/simple-sddm/Components/VirtualKeyboard.qml:26:1: module "QtQuick.VirtualKeyboard" is not installed
Jan 03 10:20:07 arch2 sddm-greeter[1045]: file:///usr/share/sddm/themes/simple-sddm/Components/Input.qml:561:5: QML Connections: Implicitly defined onFoo properties in Connections are deprecated. Use this syntax instead: function onFoo(<arguments>) { ... }
Jan 03 10:20:07 arch2 sddm-greeter[1045]: file:///usr/share/sddm/themes/simple-sddm/Components/SystemButtons.qml:70:13: Unable to assign [undefined] to QQuickItem*
Jan 03 10:20:07 arch2 sddm-greeter[1045]: file:///usr/share/sddm/themes/simple-sddm/Main.qml:35: ReferenceError: Screen is not defined
Jan 03 10:20:07 arch2 sddm-greeter[1045]: file:///usr/share/sddm/themes/simple-sddm/Components/SessionButton.qml:38:5: Unable to assign ComboBox_QMLTYPE_19 to Control_QMLTYPE_2
Jan 03 10:20:07 arch2 sddm-greeter[1045]: file:///usr/share/sddm/themes/simple-sddm/Main.qml:34: ReferenceError: Screen is not defined
Jan 03 10:20:08 arch2 sddm-greeter[1045]: Adding view for "eDP-1" QRect(0,0 1920x1080)
Jan 03 10:20:08 arch2 sddm-greeter[1045]: Message received from daemon: Capabilities
Jan 03 10:20:08 arch2 sddm-greeter[1045]: file:///usr/share/sddm/themes/simple-sddm/Components/SystemButtons.qml:70:13: Unable to assign [undefined] to QQuickItem*
Jan 03 10:20:08 arch2 sddm-greeter[1045]: file:///usr/share/sddm/themes/simple-sddm/Components/SystemButtons.qml:70:13: Unable to assign [undefined] to QQuickItem*
Jan 03 10:20:08 arch2 sddm-greeter[1045]: file:///usr/share/sddm/themes/simple-sddm/Components/SystemButtons.qml:70:13: Unable to assign [undefined] to QQuickItem*
Jan 03 10:20:08 arch2 sddm-greeter[1045]: file:///usr/share/sddm/themes/simple-sddm/Components/SystemButtons.qml:70:13: Unable to assign [undefined] to QQuickItem*
Jan 03 10:20:08 arch2 sddm-greeter[1045]: file:///usr/share/sddm/themes/simple-sddm/Components/SystemButtons.qml:70:13: Unable to assign [undefined] to QQuickItem*
Jan 03 10:20:08 arch2 sddm-greeter[1045]: file:///usr/share/sddm/themes/simple-sddm/Components/SystemButtons.qml:70:13: Unable to assign [undefined] to QQuickItem*
Jan 03 10:20:08 arch2 sddm-greeter[1045]: file:///usr/share/sddm/themes/simple-sddm/Components/SystemButtons.qml:70:13: Unable to assign [undefined] to QQuickItem*
Jan 03 10:20:08 arch2 sddm-greeter[1045]: file:///usr/share/sddm/themes/simple-sddm/Components/SystemButtons.qml:70:13: Unable to assign [undefined] to QQuickItem*
Jan 03 10:20:08 arch2 sddm-greeter[1045]: file:///usr/share/sddm/themes/simple-sddm/Components/SystemButtons.qml:70:13: Unable to assign [undefined] to QQuickItem*
Jan 03 10:20:08 arch2 sddm-greeter[1045]: file:///usr/share/sddm/themes/simple-sddm/Components/SystemButtons.qml:70:13: Unable to assign [undefined] to QQuickItem*
Jan 03 10:20:08 arch2 sddm-greeter[1045]: file:///usr/share/sddm/themes/simple-sddm/Components/SystemButtons.qml:70:13: Unable to assign [undefined] to QQuickItem*
Jan 03 10:20:08 arch2 sddm-greeter[1045]: file:///usr/share/sddm/themes/simple-sddm/Components/SystemButtons.qml:70:13: Unable to assign [undefined] to QQuickItem*
Jan 03 10:20:08 arch2 sddm-greeter[1045]: file:///usr/share/sddm/themes/simple-sddm/Components/SystemButtons.qml:70:13: Unable to assign [undefined] to QQuickItem*
Jan 03 10:20:08 arch2 sddm-greeter[1045]: file:///usr/share/sddm/themes/simple-sddm/Components/SystemButtons.qml:70:13: Unable to assign [undefined] to QQuickItem*
Jan 03 10:20:08 arch2 sddm-greeter[1045]: file:///usr/share/sddm/themes/simple-sddm/Components/SystemButtons.qml:70:13: Unable to assign [undefined] to QQuickItem*
Jan 03 10:20:08 arch2 sddm-greeter[1045]: file:///usr/share/sddm/themes/simple-sddm/Components/SystemButtons.qml:70:13: Unable to assign [undefined] to QQuickItem*
Jan 03 10:20:08 arch2 sddm-greeter[1045]: Message received from daemon: HostName
Jan 03 10:22:38 arch2 dbus-daemon[615]: [system] Activating via systemd: service name='org.freedesktop.home1' unit='dbus-org.freedesktop.home1.service' requested by ':1.37' (uid=0 pid=1277 comm="sudo systemctl status sddm")
Jan 03 10:22:40 arch2 sudo[1277]: higorlucas : TTY=tty3 ; PWD=/var/log/journal ; USER=root ; COMMAND=/usr/bin/systemctl status sddm

I've already tried only Hyprland through the terminal but it doesn't run work

I have no more ideas what it is

Computer
configuration 
CPU: AMD Ryzen 7 4800H with Radeon Graphics (16) @ 2.900GHz 
GPU: AMD ATI Radeon RX Vega 6 
GPU: NVIDIA GeForce GTX 1650 Mobile / Max-Q

**Comment by fufexan** [comment]
> I've already tried only Hyprland through the terminal but it doesn't run work

Terminal inside Xfce, or terminal as in TTY? If it doesn't work in tty, it won't work in sddm either, so just post the logs (preferrably inside a \<details> tag).

**Reply by higorlucas125** [reply]
Inside TTY, The only logs I have are from journalctl, I don't know where else to look

**Reply by fufexan** [reply]
`/tmp/hypr/<some-string>/hyprland.log` and `$XDG_CACHE_HOME/hyprland/hyprlandCrashReport<PID>.txt`

**Reply by higorlucas125** [reply]
Would it be this ?
```
[LOG] Instance Signature: d74607e414dcd16911089a6d4b6aeb661c880923_1704302873
[LOG] Hyprland PID: 48294
[LOG] ===== SYSTEM INFO: =====
[LOG] System name: Linux
[LOG] Node name: arch2
[LOG] Release: 6.1.70-1-lts
[LOG] Version: https://github.com/hyprwm/Hyprland/pull/1 SMP PREEMPT_DYNAMIC Mon, 01 Jan 2024 13:44:01 +0000

[LOG] GPU information:
01:00.0 VGA compatible controller [0300]: NVIDIA Corporation TU117M [GeForce GTX 1650 Mobile / Max-Q] [10de:1f99] (rev a1) (prog-if 00 [VGA controller])
06:00.0 VGA compatible controller [0300]: Advanced Micro Devices, Inc. [AMD/ATI] Renoir [Radeon RX Vega 6 (Ryzen 4000/5000 Mobile Series)] [1002:1636] (rev c6) (prog-if 00 [VGA controller])

[WARN] Warning: you're using an NVIDIA GPU. Make sure you follow the instructions on the wiki if anything is amiss.

[LOG] os-release:
NAME="Arch Linux"
PRETTY_NAME="Arch Linux"
ID=arch
BUILD_ID=rolling
ANSI_COLOR="38;2;23;147;209"
HOME_URL="https://archlinux.org/"
DOCUMENTATION_URL="https://wiki.archlinux.org/"
SUPPORT_URL="https://bbs.archlinux.org/"
BUG_REPORT_URL="https://bugs.archlinux.org/"
PRIVACY_POLICY_URL="https://terms.archlinux.org/docs/privacy-policy/"
LOGO=archlinux-logo

[LOG] ========================

[INFO] If you are crashing, or encounter any bugs, please consult https://wiki.hyprland.org/Crashes-and-Bugs/

[LOG]
Current splash: Take on me, take me on...

[LOG] Creating the HookSystem!
[LOG] Creating the KeybindManager!
[LOG] [hookSystem] New hook event registered: configReloaded
[LOG] Creating the AnimationManager!
[LOG] Created a bezier curve, baked 255 points, mem usage: 4.08kB, time to bake: 44.77µs. Estimated average calc time: 0.24µs.
[LOG] Creating the ConfigManager!
[LOG] NOTE: further logs to stdout / logfile are disabled by default. Use debug:disable_logs and debug:enable_stdout_logs to override this.
```

**Reply by higorlucas125** [reply]
```
--------------------------------------------
   Hyprland Crash Report
--------------------------------------------
Calm down, it was a misinput! MISINPUT!

Hyprland received signal 11 (Segmentation fault)

Version: d74607e414dcd16911089a6d4b6aeb661c880923
Tag: v0.33.1

System info:
	System name: Linux
	Node name: arch2
	Release: 6.1.70-1-lts
	Version: #1 SMP PREEMPT_DYNAMIC Mon, 01 Jan 2024 13:44:01 +0000

GPU:
	01:00.0 VGA compatible controller [0300]: NVIDIA Corporation TU117M [GeForce GTX 1650 Mobile / Max-Q] [10de:1f99] (rev a1) (prog-if 00 [VGA controller])
06:00.0 VGA compatible controller [0300]: Advanced Micro Devices, Inc. [AMD/ATI] Renoir [Radeon RX Vega 6 (Ryzen 4000/5000 Mobile Series)] [1002:1636] (rev c6) (prog-if 00 [VGA controller])


os-release:
	NAME="Arch Linux"
	PRETTY_NAME="Arch Linux"
	ID=arch
	BUILD_ID=rolling
	ANSI_COLOR="38;2;23;147;209"
	HOME_URL="https://archlinux.org/"
	DOCUMENTATION_URL="https://wiki.archlinux.org/"
	SUPPORT_URL="https://bbs.archlinux.org/"
	BUG_REPORT_URL="https://bugs.archlinux.org/"
	PRIVACY_POLICY_URL="https://terms.archlinux.org/docs/privacy-policy/"
	LOGO=archlinux-logo
	


Backtrace:
	#0 | Hyprland(_Z12getBacktracev+0x62) [0x55666b2e2b92]
		??
		??:0
	#1 | Hyprland(_ZN13CrashReporter18createAndSaveCrashEi+0x4f4) [0x55666b2b1924]
		??
		??:0
	#2 | Hyprland(_Z25handleUnrecoverableSignali+0x47) [0x55666b22cdf7]
		??
		??:0
	#3 | /usr/lib/libc.so.6(+0x3e710) [0x7fe29396f710]
		??
		??:0
	#4 | /usr/lib/libc.so.6(+0x15aedd) [0x7fe293a8bedd]
		??
		??:0
	#5 | Hyprland(+0xe3e74) [0x55666b262e74]
		??
		??:0
	#6 | Hyprland(+0x20a0bf) [0x55666b3890bf]
		??
		??:0
	#7 | Hyprland(+0xabc9a) [0x55666b22ac9a]
		??
		??:0
	#8 | Hyprland(_ZN13CInputManager21applyConfigToKeyboardEP9SKeyboard+0x911) [0x55666b324c81]
		??
		??:0
	#9 | Hyprland(_ZN13CInputManager11newKeyboardEP16wlr_input_device+0x54d) [0x55666b3260bd]
		??
		??:0
	#10 | Hyprland(_ZN6Events17listener_newInputEP11wl_listenerPv+0x137) [0x55666b2c7457]
		??
		??:0
	#11 | /usr/lib/libwayland-server.so.0(wl_signal_emit_mutable+0x7e) [0x7fe29426f01e]
		??
		??:0
	#12 | /usr/lib/libwayland-server.so.0(wl_signal_emit_mutable+0x7e) [0x7fe29426f01e]
		??
		??:0
	#13 | /usr/lib/libwlroots.so.13032(+0x507ec) [0x7fe2943087ec]
		??
		??:0
	#14 | /usr/lib/libwlroots.so.13032(+0x50aa0) [0x7fe294308aa0]
		??
		??:0
	#15 | /usr/lib/libwlroots.so.13032(+0x54dd6) [0x7fe29430cdd6]
		??
		??:0
	#16 | Hyprland(_ZN11CCompositor15startCompositorEv+0x58e) [0x55666b22e07e]
		??
		??:0
	#17 | Hyprland(main+0xa89) [0x55666b216009]
		??
		??:0
	#18 | /usr/lib/libc.so.6(+0x27cd0) [0x7fe293958cd0]
		??
		??:0
	#19 | /usr/lib/libc.so.6(__libc_start_main+0x8a) [0x7fe293958d8a]
		??
		??:0
	#20 | Hyprland(_start+0x25) [0x55666b22a035]
		??
		??:0


Log tail:
 Registered signal for owner 55666c0258f0: 55666c0259e8 -> 55666b401400 (owner: WLRCursor)
[LOG] Registered signal for owner 55666c0258f0: 55666c0259d8 -> 55666b4013e0 (owner: WLRCursor)
[LOG] Registered signal for owner 55666c0258f0: 55666c0259f8 -> 55666b4013c0 (owner: WLRCursor)
[LOG] Registered signal for owner 55666c0258f0: 55666c025a18 -> 55666b4013a0 (owner: WLRCursor)
[LOG] Registered signal for owner 55666c0258f0: 55666c0259b8 -> 55666b401380 (owner: WLRCursor)
[LOG] Registered signal for owner 55666c0258f0: 55666c0259c8 -> 55666b401360 (owner: WLRCursor)
[LOG] Registered signal for owner 55666b5ddc10: 55666b5ddc28 -> 55666b401340 (owner: Backend)
[LOG] Registered signal for owner 55666b5b89e8: 55666c133d18 -> 55666b401320 (owner: Seat)
[LOG] Registered signal for owner 55666b5b89e8: 55666c133d28 -> 55666b401300 (owner: Seat)
[LOG] Registered signal for owner 55666b5b89e8: 55666c133d68 -> 55666b4012e0 (owner: Seat)
[LOG] Registered signal for owner 55666b5b89e8: 55666c133d78 -> 55666b4012c0 (owner: Seat)
[LOG] Registered signal for owner 55666b5b89e8: 55666c133d28 -> 55666b401300 (owner: Seat)
[LOG] Registered signal for owner 55666b5b89e8: 55666c133d48 -> 55666b4012a0 (owner: Seat)
[LOG] Registered signal for owner 55666c133f80: 55666c133fa0 -> 55666b401280 (owner: LayerShell)
[LOG] Registered signal for owner 55666c0256e0: 55666c025708 -> 55666b401260 (owner: OutputLayout)
[LOG] Registered signal for owner 55666c1341a0: 55666c1341d8 -> 55666b401240 (owner: OutputMgr)
[LOG] Registered signal for owner 55666c1341a0: 55666c1341e8 -> 55666b401220 (owner: OutputMgr)
[LOG] Registered signal for owner 55666c025bd0: 55666c025be8 -> 55666b401200 (owner: PointerConstraints)
[LOG] Registered signal for owner 55666c1340f0: 55666c134120 -> 55666b4011e0 (owner: XDGDecoMgr)
[LOG] Registered signal for owner 55666c025dd0: 55666c025e00 -> 55666b4011c0 (owner: VirtPtrMgr)
[LOG] Registered signal for owner 55666c025d20: 55666c025d50 -> 55666b4011a0 (owner: VKeyboardMgr)
[LOG] Registered signal for owner 55666b88b790: 55666b88b790 -> 55666b401180 (owner: WLRRenderer)
[LOG] Registered signal for owner 55666c026160: 55666c026190 -> 55666b401160 (owner: WLRIdleInhibitMgr)
[LOG] Registered signal for owner 55666c025750: 55666c025780 -> 55666b401140 (owner: PowerMgr)
[LOG] Registered signal for owner 55666c0265e0: 55666c026610 -> 55666b401120 (owner: IMEMgr)
[LOG] Registered signal for owner 55666c026530: 55666c026560 -> 55666b401100 (owner: TextInputMgr)
[LOG] Registered signal for owner 55666c026690: 55666c0266b8 -> 55666b4010e0 (owner: ActivationV1)
[LOG] Registered signal for owner 55666c0267e0: 55666c0267e8 -> 55666b4010c0 (owner: SessionLockMgr)
[LOG] Registered signal for owner 55666c025630: 55666c025670 -> 55666b4010a0 (owner: GammaCtrlMgr)
[LOG] Registered signal for owner 55666c026880: 55666c026888 -> 55666b401080 (owner: CursorShapeMgr)
[LOG] Registered signal for owner 55666c026920: 55666c026950 -> 55666b401060 (owner: TearingControlMgr)
[LOG] Registered signal for owner 55666b5b8750: 55666c025f70 -> 55666b401040 (owner: DRM)
[LOG] Registered signal for owner 55666b5ddca0: 55666b5dde00 -> 55666b401020 (owner: Session)
[LOG] wl_display_add_socket for wayland-1 succeeded with 0
[LOG] Running on WAYLAND_DISPLAY: wayland-1
[LOG] Attached a keyboard with name Acer Wireless Radio Control
[LOG] Registered signal for owner 55666c1f3160: 55666c19bd50 -> 55666c1f3168 (owner: Keyboard)
[LOG] Registered signal for owner 55666c1f3160: 55666c19bd40 -> 55666c1f31d0 (owner: Keyboard)
[LOG] Registered signal for owner 55666c1f3160: 55666c19bc20 -> 55666c1f32a0 (owner: Keyboard)
[LOG] Registered signal for owner 55666c1f3160: 55666c19bd60 -> 55666c1f3238 (owner: Keyboard)
[LOG] ApplyConfigToKeyboard for "acer-wireless-radio-control", hasconfig: 0
[LOG] Attempting to create a keymap for layout (unset) with variant  (rules: , model: , options: grp:alt_shift_toggle)
[ERR] Keyboard layout (unset) with variant  (rules: , model: , options: grp:alt_shift_toggle) couldn't have been loaded.
[LOG] [hookSystem] New hook event registered: activeLayout

```

**Reply by higorlucas125** [reply]
cat /etc/mkinitcpio.conf
```
# vim:set ft=sh
# MODULES
# The following modules are loaded before any boot hooks are
# run.  Advanced users may wish to specify all system modules
# in this array.  For instance:
#     MODULES=(usbhid xhci_hcd)
MODULES=(nvidia nvidia_modeset nvidia_uvm nvidia_drm)

```
